x-pack/plugins/index_management/common/types/indices.ts

```diff
@@ -5,29 +5,9 @@
  * 2.0.
  */
 
-export type { Index } from '@kbn/index-management-shared-types';
+import { IndicesIndexSettingsKeys } from '@elastic/elasticsearch/lib/api/types';
 
-export interface IndexModule {
```

Comment:
Removing this manual type in favor of `IndicesIndexSettingsKeys` type from Es client.